spec/models/configuration_script_base_spec.rb

```diff
@@ -0,0 +1,13 @@
+describe ConfigurationScriptBase do
+  it "has many authentications and vice versa" do
```

Comment:
Funny that we are now discussing about testing inflection, which was *never* my intention.   This spec reads `it "has many authentications and vice versa"`. That was my intention with this spec to begin with.  I had no intention to test inflection or any other Rails guts. I am simply asserting the relationship to be maintained.